src/cls/lock/cls_lock.cc

```diff
@@ -297,6 +297,8 @@ static int remove_lock(cls_method_context_t hctx,
   // remove named locker from set
   map<locker_id_t, locker_info_t>::iterator iter = lockers.find(id);
   if (iter == lockers.end()) { // no such key
+    CLS_LOG(10, "locker %s [name: %s.%ld, cookie: %s] did not exist", name.c_str(), 
```

Comment:
s/did/does/

Reply:
@dillaman could you take a quick look? as i think rbd is the only major user of `cls::lock`.

Reply:
Rados striper will use cls::lock too.